plugins/woocommerce/includes/rest-api/Controllers/Version3/class-wc-rest-orders-controller.php

```diff
@@ -219,7 +219,8 @@ protected function save_object( $request, $creating = false ) {
 
 			// Set status.
 			if ( ! empty( $request['status'] ) ) {
-				$object->set_status( $request['status'] );
+				$manual_update = isset( $request['manual_update'] ) ? $request['manual_update'] : false;
```

Comment:
The `save_object` method has a boolean `$creating` parameter, which makes me wonder, should we allow the possibility of doing a manual update when _creating_ an order, or only when updating an existing order? If it's only for updating existing orders, we should probably add a check here to make sure `$creating` is `false`.

Reply:
I guess I would lean towards allowing it on order creation as well, but I'm curious what you think.